packages/components/src/ListView/Items/Items.component.js

```diff
@@ -99,7 +99,10 @@ class Items extends React.PureComponent {
 	}
 
 	renderItem(item, index) {
-		const computedIndex = this.hasToggleAll ? index + 1 : index;
+		let computedIndex = this.hasToggleAll ? index + 1 : index;
```

Comment:
i think this is really hight on id collision chance, even with `item` appended, i would not allow to render an id if the component user didn't provide an prefix or postfix id fragment